src/app/wallet/wallet.component.ts

```diff
@@ -1,6 +1,6 @@
 import { Component, OnInit } from '@angular/core';
```

Comment:
Why is the [transaction list feature](https://github.com/AMPnet/ampnet-crowdfunding-frontend/commit/b7f6392a0a8a6bad9e7147a50cfaa3c90bb03184) (merged earlier) removed from this pull request?
Be careful when resolving merge issues (conflicts). If something is not clear, ask the author of conflicting lines of code (git blame, editor helps you).

Reply:
It shouldn't be removed. I have the newest version of this branch and all the features of the tx list are implemented.